bodhi/models/models.py

```diff
@@ -2007,6 +2008,9 @@ class User(Base):
     name = Column(Unicode(64), unique=True, nullable=False)
     email = Column(UnicodeText, unique=True)
 
+    # A preference
+    show_popups = Column(Boolean, default=True, server_default=text('TRUE'))
```

Comment:
Yeah, I tried ``server_default=True`` first and it balked.  I tried this a few times with upgrades and downgrades.